internal/schema/framework_identifier.go

```diff
@@ -42,6 +42,26 @@ func (identifier FrameworkIdentifier) ToCamelCase() string {
 	return string(unicode.ToLower(firstLetter)) + pascal[size:]
 }
 
+// ToPrefixCamelCase will return a camel case formatted string of the identifier,
+// prefixed with a camel-cased version of the supplied name if the identifier is
+// a generated custom value method name.
+// Example:
+//   - equal(something) -> somethingEqual
+//   - type(something) -> somethingType
+func (identifier FrameworkIdentifier) ToPrefixCamelCase(name string) string {
```

Comment:
nit: Would changing the name of the input parameter to `prefix` make more sense in this context? Same for the method below

Reply:
Fair point. Have updated.